ipa-core/src/utils/vec_chunks.rs

```diff
@@ -0,0 +1,58 @@
+use std::cmp::min;
+
+pub struct VecChunks<T: Clone> {
+    vec: Vec<T>,
+    pos: usize,
+    chunk_size: usize,
+}
+
+impl<T: Clone> Iterator for VecChunks<T> {
+    type Item = Vec<T>;
```

Comment:
I checked the usage and it seems that you could yield `&[u8]`. Maybe there are plans to use owned chunks later?

Reply:
I started from "aggregate_values takes a stream of owned values" and ended up with this, but the reasoning wasn't sound, getting `&[T]` from `slice::chunks` is sufficient.